### PR TITLE
Add 'participant-' prefix to name and id inputs for clarity

### LINKED ISSUE #327: Clarify the Id for Pools and plain processes
__Is your feature request related to a problem? Please describe__

For beginners it is confusing that the process id is handled in different fields depending if you modeled a process with a pool or without.

If you model a process without a pool, you have a Id and a Name to fill:
grafik

If you model a process inside a pool, you have to fill a Process Id and a Process Name:
grafik

__Describe the solution you'd like__

Rename the labels: 

* Process Id and Process Name for the first picture.
* Participant Id, Participant Name, Process Id and Process Name for the second picture.

__Describe alternatives you've considered__

Keep it as it is.

__Additional context__

This will help beginners in trainings a lot, as it is confusing that the same process needs different fields if there is pool or not. The behaviour of the engine and the Cockpit is easier to understand.

### PR DESCRIPTION
**Pull request by azeghers**
__Which issue does this PR address?__

Closes #327

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by azeghers** [comment]
@andreasgeier 

This is how the properties panel looks for a participant :

image

And this is how it looks for any other element :

image

File: lib/provider/bpmn/parts/IdProps.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function(group, element, translate, options) {
   // Id
   group.entries.push(entryFactory.validationAwareTextField(translate, {
     id: options.id || 'id',
-    label: translate('Id'),
+    label: options.label || translate('Id'),
```
Comment (nikku) [inline lib/provider/bpmn/parts/IdProps.js:18]: What about translating here, as we already translate the description as part of the `IdProps`, too.

**Review by andreasgeier — APPROVED** [review]
Looks as expected

File: lib/provider/bpmn/parts/IdProps.js (hunk 1 above)
Comment (azeghers) [reply lib/provider/bpmn/parts/IdProps.js:18]: The `option.label` attribute is defined in the `getIdOptions` of the properties provider as such :
```javascript
    return { id: 'participant-id', label: translate('Participant Id') };
```
So it already should be translated by the time it ends up in this function.
 (I don't know how to link to a file inside of the PR)

Comment (azeghers) [reply lib/provider/bpmn/parts/IdProps.js:18]: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/412/files#diff-9578e9e6d7be08c860d340b59799ea827673c391d9163b522c8199e7e9a5515fR135-R140

Comment (nikku) [reply lib/provider/bpmn/parts/IdProps.js:18]: Got it. So the assumption is: If something is passed, it is always translated already.

Comment (nikku) [reply lib/provider/bpmn/parts/IdProps.js:18]: However, we handle the `description` props differently.

These are translated, after the have been passed, cf. https://github.com/bpmn-io/bpmn-js-properties-panel/blob/e9f0346513f333d8804ab822a9a1a615835e531a/lib/provider/bpmn/parts/IdProps.js#L13.

Comment (nikku) [reply lib/provider/bpmn/parts/IdProps.js:18]: My proposal would be: We translate consistently, inside `*Props`, always.

Comment (nikku) [reply lib/provider/bpmn/parts/IdProps.js:18]: Alternative: Consistently translate _outside_ the `*Props`, always.

**Pull request merged**